vdr/api/v1/api.go

```diff
@@ -122,12 +122,12 @@ func (a *Wrapper) CreateDID(ctx context.Context, request CreateDIDRequestObject)
 			if err != nil {
 				return nil, core.InvalidInputError("controller entry (%s) could not be parsed: %w", c, err)
 			}
-			options.Controllers = append(options.Controllers, *id)
+			options.AddOption(didnuts.Controllers(*id))
```

Comment:
this adds a controllers option per controller instead of one option for all controllers. It works, but cleaner to add the option outside the loop imo